features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/index.js

```diff
@@ -95,14 +95,12 @@ class Details extends React.Component {
         let existing_subscriptions = api.getSubscriptions(this.api_uuid, null);
         let promised_applications = api.getAllApplications();
 
-        //Promise.all([ promised_api, existing_subscriptions, promised_applications] ).then(
-        Promise.all([ promised_api] ).then(
+        Promise.all([ promised_api, existing_subscriptions, promised_applications] ).then(
             response => {
-                //let [api, subscriptions, applications] = response.map(data => data.obj);
-                let [api] = response.map(data => data.obj);
+                let [api, subscriptions, applications] = response.map(data => data.obj);
                 //Getting the policies from api details
                 this.setState({api: api});
-                /*if(api && api.policies){
+                if(api && api.policies){
```

Comment:
Please fix the formatting issue.

Reply:
The file has been updated with new logic and replaced with [index.jsx](https://github.com/wso2/carbon-apimgt/blame/6.5.x/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/index.jsx)